src/arguments.cpp

```diff
@@ -422,6 +423,10 @@ Error Arguments::parse(const char* args) {
             CASE("reverse")
                 _reverse = true;
 
+            CASE("target-cpu")
+                if (value == NULL || (_target_cpu = atoi(value)) < 0)
+                    _target_cpu = -1;
```

Comment:
Always surround `if` body with `{ }`. Exception is trivial one-liners like `if (!cond) return;`

Reply:
e6ef966dd053ab35b30f51fdc50cfd7002ee2370